client/asset/btc/spv_wrapper.go

```diff
@@ -191,15 +191,44 @@ var (
 // logRotator initializes a rotating file logger.
 func logRotator(dir string) (*rotator.Rotator, error) {
 	const maxLogRolls = 8
-	logDir := filepath.Join(dir, logDirName)
-	if err := os.MkdirAll(logDir, 0744); err != nil {
-		return nil, fmt.Errorf("error creating log directory: %w", err)
+	logDir := filepath.Join(filepath.Dir(dir), logDirName)
+	exist, err := fileExists(logDir)
+	if err != nil {
+		return nil, err
+	}
+	if !exist {
+		oldDir := filepath.Join(dir, "logs")
+		exist, err := fileExists(oldDir)
+		if err != nil {
+			return nil, err
+		}
+		if !exist {
+			if err := os.MkdirAll(logDir, 0744); err != nil {
+				return nil, fmt.Errorf("error creating log directory: %w", err)
+			}
+		} else {
+			// Move old logs to new log path.
+			if err := os.Rename(oldDir, logDir); err != nil {
+				return nil, err
+			}
+		}
```

Comment:
I personally think it'd be fine to skip this "upgrade" step.

Reply:
Was thinking the same, especially since it's a perpetual upgrade.  I'd want to remove this stuff eventually, but when?  Let's just leave things

Reply:
>  Let's just leave the old log alone

Alright.